test/test_evaluation.py

```diff
@@ -46,7 +46,7 @@
         ),
     ],
 )
-def check_evaluation(str_expr: str, str_expected: str, message=""):
```

Comment:
This was a plain bug I introduced a while a go. The name has to start with `test_` in order for `py.test` to notice it is a test case.